tutorial/scripts/LFRic/distributed_memory/4_reductions/README.md

```diff
@@ -0,0 +1,66 @@
+In this example you will add a builtin that performs a reduction to
```

Comment:
"...will edit the algorithm layer to additionally call a builtin that..."

Reply:
Rewrote the words as I was not happy with them.